developer/src/kmc-kmn/src/kmw-compiler/validate-layout-file.ts

```diff
@@ -45,6 +45,14 @@ function KeyIdType(FId: string): TKeyIdType {   // I4142
     if(VKeyNames.find(key => key.toUpperCase() == FId)) {
       return TKeyIdType.Key_Constant;
     }
+
+    if(KeymanWebTouchStandardKeyNames.indexOf(FId) >= 0) {
+      return TKeyIdType.Key_Constant;
+    }
+
+    if(KMWAdditionalKeyNames.indexOf(FId) >= 0) {
+      return TKeyIdType.Key_Constant;
+    }
```

Comment:
These were added to fix compile warnings that were being emitted for keys that were actually valid...